package.json

```diff
@@ -151,13 +149,15 @@
     "simplehttp2server": "^2.0.0",
     "source-map": "^0.5.6",
     "stack-trace": "0.0.10",
+    "style-loader": "^0.18.2",
     "sw-precache-webpack-plugin": "^0.11.2",
     "tmp": "0.0.31",
     "unfetch": "^3.0.0",
-    "url-loader": "^0.5.8",
-    "webpack": "^2.3.3",
+    "url-loader": "^0.5.9",
+    "webpack": "^3.3.0",
```

Comment:
I'd definitely recommend using something higher than 3.3.0 if you're going to use `ModuleConcatenationPlugin`. I had to back out the use of the plugin in an internal app using 3.3.0 due to naming collisions and other strange demons. Check out the release notes of every release since 3.3.0 and you'll see module concat bugfixes. 👍 https://github.com/webpack/webpack/releases